dotCMS/src/main/java/com/dotcms/contenttype/business/RelationshipFactoryImpl.java

```diff
@@ -383,6 +385,21 @@ public  Relationship dbByTypeValue(final String typeValue){
         return matches;
     }
 
+    @Override
+    public  boolean isParent(final Relationship relationship, final Field field) {
```

Comment:
Are the `isParent` and `isChild` methods named correctly?  Because it looks to me if you call isParent on with a  relationship and a child field, it returns true?  It's a test on the field, right?  Shouldn't isParent return false when passed the child field?

Reply:
I agree. I have renamed those method and registered a [new card](https://github.com/dotCMS/core/issues/17564) to refactor our logic to always use isChildField and isParentField instead of isParent and isChild